src/EA.Weee.RequestHandlers.Tests.Unit/AatfEvidence/GetAatfSummaryRequestHandlerTests.cs

```diff
@@ -37,7 +37,7 @@ public GetAatfSummaryRequestHandlerTests()
             mapper = A.Fake<IMapper>();
             evidenceStoredProcedures = A.Fake<IEvidenceStoredProcedures>();
 
-            request = new GetAatfSummaryRequest(fixture.Create<Guid>());
+            request = new GetAatfSummaryRequest(fixture.Create<Guid>(), 2022);  //TODO: check this
```

Comment:
similar comment to other, create private readonly var for ComplianceYear and use fixture to initialise and pass to the constructor. or just do fixture.Create<int> rather than hard coding 2022